### DISCUSSION TITLE
Anonymous DNSCrypt relays

### DISCUSSION THREAD

**Discussion by erns8** [AdguardTeam/AdGuardHome#3891] [Ideas]
Hi, I love Adguard Home, I would like to share my ideas with you. The first one is the possibility to use [anonymous DNSCrypt relays](https://github.com/DNSCrypt/dnscrypt-resolvers/blob/master/v3/relays.md) in the DNS settings [(exemple)](https://www.meganerd.nl/encrypted-dns-server) . The second is to be able to add blocklists only for specific clients, and finally the addition of Google in the "blocked services" section. Thanks for your hard work and have a nice weekend.

**Comment by ainar-g** [comment]
Hello and thanks for the suggestions.

 >  add blocklists only for specific clients

This feature request will be addressed in #2554.

 >  the addition of Google in the "blocked services" section

There is #2604 and a few similar requests.
